packages/webpack-utilities/package.json

```diff
@@ -19,16 +19,16 @@
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.0.0-beta.0",
     "@types/loader-utils": "1.1.3",
-    "@types/webpack": "4.4.0",
+    "@types/webpack": "4.41.26",
     "@fluentui/scripts": "^1.0.0",
-    "webpack": "4.43.0"
+    "webpack": "^5.12.3"
   },
   "dependencies": {
     "loader-utils": "^1.1.0",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
-    "webpack": ">=4.0.0",
+    "webpack": ">=5.0.0",
```

Comment:
Does this need to be bumped or is 4 still okay as a peer?

Reply:
hmmm; I don't know. I don't know what this is used for. I'll poke around and see.

Reply:
I looked into this a little the other day and found a bunch of fabricAsyncLoader references in OOUI

Reply:
I think it should likely accept either 4 or 5, but i'll ask ken.